package.json

```diff
@@ -862,6 +906,18 @@
         "fileMatch": "esy.json",
         "url": "https://raw.githubusercontent.com/esy/esy-schema/master/esySchema.json"
       }
+    ],
+    "customEditors": [
+      {
+        "viewType": "ast-editor",
+        "displayName": "OCaml AST preview",
+        "priority": "option",
+        "selector": [
+          {
+            "filenamePattern": "*.ml"
```

Comment:
I'm not familiar with this option, but why is `mli` excluded here?

Reply:
I missed it since I added `mli` support only lately. Surprisingly, I didn't break when opening AST explorer with: 
```Ocaml 
Vscode.Commands.executeCommand ~command:"vscode.openWith"
      ~args:
        [ Uri.t_to_js uri
        ; Ojs.string_to_js "ast-editor"
        ; ViewColumn.t_to_js ViewColumn.Beside
        ]
``` 
Added with 8066285d1e8a3a41ea89b6eb3f59712d074da8e0 so it's there.

Reply:
Oops, i somehow messed it up, it's actually in 2fc5ebd2f54cfe2a72e86b44670d2243f824cacf